rfcs/text/0000-analytics-instrumentation-plan.md

```diff
@@ -0,0 +1,137 @@
+### DCP PR:
+
+# Analytics instrumentation plan
+
+## Summary
+
+To support collection of the Human Cell Altas (HCA) Data Coordination Platform (DCP) user metrics desired by the DCP
+development team, the HCA DCP service must be instrumented with a user tracking solution. This document describes the
+goals for the tracking solution and a proposal for instrumenting HCA DCP service components.
+
+## Author(s)
+
+* [Matt Weiden](mailto:mweiden@chanzuckerberg.com)
+* [Marcio Von Muhlen](mailto:mvonmuhlen@chanzuckerberg.com)
+* [Adrienne Sussman](mailto:%20asussman@chanzuckerberg.com)
+
+## Shepherd
+
+## Motivation
+
+The development team needs quantitative data on application usage to drive informed decisions on how HCA DCP should
+evolve as a product. The following goals address this need.
```

Comment:
The motivation section includes goals (which sound great) but I am not sure have had legal guidance. I am not sure anyone in the DCP implementation team has this type of training, as well, the HCA is in the process of evaluating how to respond to GDPR. This should be informed by that work.

Reply:
Note: This is not to stop this plan but to indicate it needs to be reviewed by the legal team looking into GDPR before implementation. I am happy to get it to them when you are ready.

Reply:
Update: This has been reviewed by the GDPR Task Force. Resolving this.